mantidimaging/gui/windows/spectrum_viewer/model.py

```diff
@@ -195,6 +196,28 @@ def save_csv(self, path: Path, normalized: bool) -> None:
             csv_output.write(outfile)
             self.save_roi_coords(self.get_roi_coords_filename(path))
 
+    def save_rits(self, path: Path, normalized: bool) -> None:
+        """
+        Saves the spectrum for one ROI to a RITS file.
+
+        @param path: The path to save the CSV file to.
+        @param normalized: Whether to save the normalized spectrum.
+        """
+        if self._stack is None:
+            raise ValueError("No stack selected")
+
+        # Default_roi will likely need updating once UI is implemented
+        default_roi = self.default_roi_list[0]
+        tof = np.arange(self._stack.data.shape[0])
+        transmission_error = self.get_spectrum(default_roi, SpecType.SAMPLE)
+        if normalized:
+            if self._normalise_stack is None:
+                raise RuntimeError("No normalisation stack selected")
+            transmission = self.get_spectrum(default_roi, SpecType.OPEN)
+        else:
+            transmission = self.get_spectrum(default_roi, SpecType.SAMPLE)
```

Comment:
If RITS wants "transmission", then I think we only need the `normalised` case. I.e. transmission = sample/open.